test/quantization/test_quantize_script.py

```diff
@@ -1371,7 +1352,8 @@ def forward(self, x):
                              (NonQuantizedAddScalar(), False),
                              (NonQuantizedInplaceAddScalar(), False)]:
             op = "quantized::add_scalar" if quantized else "aten::add"
-            m = self._test_op_impl(m, data, op)
+            # TODO: fix debug=True numerics
+            m = self._test_op_impl(m, data, op, check=False)
```

Comment:
All the op impls call the same model m, which gets called with quantize_script twice. Can we refactor the tests so that we call quantize_script on the traced and scripted model with debug=True/False and then just check for the quantized op being present/absent?

Reply:
adding tracing support will be a separate effort as well, maybe we'll need more help to cover tracing on all ops

Reply:
In that case, lets add support for scripting first and refactor the test. Let make a list for what ops need tracing support and follow up in later PRs and track the work.